config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-release-v1beta1.yaml

```diff
@@ -102,7 +102,7 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: capz-periodic-capi-e2e-v1beta1
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
-- name: periodic-cluster-api-provider-azure-e2e-full-v1beta1
+- name: periodic-cluster-api-provider-azure-e2e-v1beta1
```

Comment:
We also have v1beta1-minus-1 don't we? If so we should update it too so all branches are equivalent

Reply:
Yes. Not sure what you mean, changes to the "minus-1" job(s) are in this file.

Reply:
I was looking for a separate file

Reply:
Got it, so does that mean we're good here, and the final step before this merges is container consensus?

Reply:
Indeed.